webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Parameters/ParametersTable.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { TableComposable, Thead, Tbody, Tr, Th } from '@patternfly/react-table';
+import {
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+  Button,
+  Spinner,
+  SearchInput,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../../../common/I18n';
+import { ParametersTableRow } from './TableRow';
+import { columnNames, HOST_PARAM } from './ParametersConstants';
+import Pagination from '../../../Pagination';
+import { useForemanSettings } from '../../../../Root/Context/ForemanContext';
+import './Parameters.scss';
+import { STATUS } from '../../../../constants';
+
+export const ParametersTable = ({
+  status,
+  hostId,
+  allParameters,
+  editHostsPermission,
+}) => {
+  const [editingRow, setEditingRow] = useState(-1);
+  const [activeSortDirection, setActiveSortDirection] = useState('asc');
+  const { perPage: settingsPerPage = 20 } = useForemanSettings() || {};
```

Comment:
Do we need to hardcode a default here? I don't like the magic numbers everywhere. If we do decide to change the default, we need to go through the whole codebase.